### ISSUE TITLE
For the first transaction of some addresses, the account balance of Before is not 0

### ISSUE THREAD

**Issue by yzh-d** [blockscout/blockscout#8910]
### Description

Check the transactions of some account addresses and find that the account balance of Before is not 0.

For example, the address: 0x92CB244Ded92A32AD938D680f1b0EB2A87182D39, only one transaction is generated, and the transaction hash is 0x902530f1e4f9027eccde31da240132f0026425f08259ddc09ecea327eb2b73b1. View the state in the transaction details as follows
[https://scan.aries.axiomesh.io/tx/0x902530f1e4f9027eccde31da240132f0026425f08259ddc09ecea327eb2b73b1](url)
<img width="1284" alt="image" src="https://github.com/blockscout/blockscout/assets/30540477/74fa7ac2-17a7-4005-8f7b-e6b117ed72b0">

For example, address: 0x73c45D3726292F0965e177a365Ff461114b42D3c. You can check that the first two transactions also have similar problems.
- first transaction

  - [https://scan.aries.axiomesh.io/tx/0x17d3f9205bb013c84744edf190c99c85741b205ff535b1b1c272cedce8dd3816](url)
  - 
<img width="1238" alt="image" src="https://github.com/blockscout/blockscout/assets/30540477/09b559c3-966a-4c7b-bb5e-f295e3373f51">

- second transaction

  - [https://scan.aries.axiomesh.io/tx/0xb0f60934d757b01e18261c31b0b70e5e07c16ce1c994aa59fa32844b02428fd2](url)
  - 
<img width="1169" alt="image" src="https://github.com/blockscout/blockscout/assets/30540477/be9da53d-4039-4666-84e3-50581311bd6f">

### Type of the installation

Docker-compose

### Type of the JSON RPC archive node

ganache

### Type of the chain

L1

### Link to the page

https://scan.aries.axiomesh.io/tx/0xb0f60934d757b01e18261c31b0b70e5e07c16ce1c994aa59fa32844b02428fd2

### Backend version

v5.2.3-beta

### Frontend version

Self-developed front-end

### Elixir & Erlang/OTP versions

Elixir 1.14.0 (compiled with Erlang/OTP 25)

### Operating system

Linux

**Comment by ekawibisono** [comment]
hi @yzh-d 
So your problem is in blockscout or in AXMScan?

**Comment by yzh-d** [comment]
> hi @yzh-d So your problem is in blockscout or in AXMScan?
> 
> I don't understand your problem. I checked the link does not use blockscout explorer.

The problem is that in blockscout, the front-end is our own, and the back-end version is the v5.2.3-beta version of blockscout used.

**Comment by ekawibisono** [comment]
ah ok. you use backend blockscout right ?, just waiting for team core reply your issue

**Comment by yzh-d** [comment]
> ah ok. you use backend blockscout right ?, just waiting for team core reply your issue

Yes, we use backend blockscout, this version is v5.2.3-beta, just like the question I mentioned above, this should be a defect, hope to get an official reply

**Comment by nikitosing** [comment]
Closing the issue. If it is still reproducible with the new Blockscout version, please create a new issue.
